### PR TITLE
Dsm checkstyle

### PR DESCRIPTION
**Pull request by SimoneMaiwald**
## Context

more Checkstyle fixes

### CODE REVIEW

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java [hunk 1]
```diff
@@ -23,15 +23,15 @@
     String upsTrackingDate;
     String upsReturnStatus;
     String upsReturnDate;
-    String HRUID;
+    String hruid;
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java:26]: @pegahtah ok to change the name here or is it used in a response/ui somewhere?

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java [hunk 2]
```diff
@@ -23,15 +23,15 @@
...
     String externalOrderNumber;
-    boolean CEOrdered;
+    boolean ceOrdered;
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java:28]: @pegahtah ok to change the name here or is it used in a response/ui somewhere?

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java [hunk 3]
```diff
@@ -44,14 +44,13 @@ public DdpKit(String dsmKitRequestId, String kitLabel, String trackingToId, Stri
         this.upsTrackingDate = upsTrackingDate;
         this.upsReturnStatus = upsReturnStatus;
         this.upsReturnDate = upsReturnDate;
-        this.HRUID = bspCollaboratodId;
+        this.hruid = bspCollaboratorId;
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java:47]: @pegahtah does that make sense? bspCollaboratorId != hruid for most customers

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/ups/UPSKit.java [hunk 4]
```diff
@@ -15,7 +15,7 @@
     // the two shipments differ in tracking id
     UPSPackage upsPackage; // package with tracking_to_id
     String kitLabel;
-    Boolean CE_order;
+    Boolean ceOrder;
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/ups/UPSKit.java:18]: @pegahtah ok to change the name here or is it used in a response/ui somewhere?

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java (hunk 3 above)
Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java:47]: The class was only for Tbos  cloud function

**Review by Trimax — DISMISSED** [review]

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java (hunk 1 above)
Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java:26]: I think this is ok

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java (hunk 2 above)
Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java:28]: no not used in pubsub or anything like that as far as I know

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/ups/UPSKit.java (hunk 4 above)
Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/ups/UPSKit.java:18]: yes

**Review by pegahtah — APPROVED** [review]

**Pull request merged**